Description: Ensure internal headers are picked instead of system headers.
Origin: http://anonscm.debian.org/gitweb/?p=collab-maint/afpfs-ng.git;a=blob;f=debian/patches/header-path-fix.patch

--- a/include/afp.h
+++ b/include/afp.h
@@ -7,8 +7,8 @@
 #include <netdb.h>
 #include <sys/statvfs.h>
 #include <pwd.h>
-#include <afp_protocol.h>
-#include <libafpclient.h>
+#include "afp_protocol.h"
+#include "libafpclient.h"
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
